solutions/form-builder/src/app/app.module.ts

```diff
@@ -0,0 +1,21 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    FormsModule,
```

Comment:
Oops, good catch. Forgot to remove that when I added `ReactiveFormsModule`. Removed 👍